### PR TITLE
lekse da dixe

### PR DESCRIPTION
**Pull request by jkunimune15**
ca mi suje sa predi mes.

### CODE REVIEW

File: pandunia-lekse.csv [hunk 1]
```diff
@@ -515,8 +517,11 @@ dominogem||||dominoes||dominó||||||||||||||dominopeli|domino
 domu||||live (dwell, reside)|vivre (habiter)|vivir (residir)|||||||||||||loĝi|asua|mieszkać
 dona|||eng:dan:give, deu:geben, zho:给 (gěi)|give||dar|||||||||||||doni|antaa|dać, dawać
 done||||gift||regalo (obsequio)|||||||||||||donaco|lahja|podarunek, dar, prezent
-dong|||zho:东 (dōng), yue:東 (dōng), vie:đông, kor:동 (dong)|east||éste|||||||||||||oriento|itä|wschód
+donge|||zho:东 (dōng), yue:東 (dōng), vie:đông, kor:동 (dong)|east||éste|||||||||||||oriento|itä|wschód
+dongi||||eastern||oriental|||||||||||||||
+dongia||||longitude||longitud|||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: ye no sahi. ablo "westodongia" funcu max bono.

File: pandunia-lekse.csv [hunk 2]
```diff
@@ -515,8 +517,11 @@ dominogem||||dominoes||dominó||||||||||||||dominopeli|domino
...
 dongi timoria|dex|TL||East Timor (Timor-Leste)||Timor Oriental||||||||||||||Itä-Timor|
+dongo||||eastward||al este|||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: espani misale xowa: yi lekse no sahi. le xudu esa "pa donge", eska no?

File: pandunia-lekse.csv [hunk 3]
```diff
@@ -785,7 +790,13 @@ gati|||tel:గట్టి (gaṭṭi), tur:katı, hin:कड़ा (kaṛā), j
 gatomate||||solid||sólido||||固体|固体||||||||solido|kiinteä aine|
 gatomati||||rigid (solid)||rígido (sólido)||||||||||||||kiinteä|
 gawa||||raise (lift)||levantar (elevar)||||||||||||||nostaa|
+gawe||||top||alto (parte de arriba)|||||||||||||||
 gawi|||zho:高 (gāo), vie:cao|high||alto|||||||||||||alta|korkea|wysoki
+gawia||||height (altitude)||altura (altitud)|||||||||||||||
+gawo||||up||arriba|||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: pa gawe

File: pandunia-lekse.csv [hunk 4]
```diff
@@ -785,7 +790,13 @@ gati|||tel:గట్టి (gaṭṭi), tur:katı, hin:कड़ा (kaṛā), j
...
+gawodarja||||upgrade (level up)||mejorar|||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: eska "bonodarja" u "darjobona"?

File: pandunia-lekse.csv [hunk 5]
```diff
@@ -785,7 +790,13 @@ gati|||tel:గట్టి (gaṭṭi), tur:katı, hin:कड़ा (kaṛā), j
...
+gawolada||||upload||cargar (subir)|||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: netopoza?

File: pandunia-lekse.csv [hunk 6]
```diff
@@ -1158,9 +1178,10 @@ karawe|||ara:(karāwiya), eng:caraway, fra:carvi, spa:alcaravea, por:alcaravia|c
 karbone|mate|C||carbon (coal)||carbón|||||||||||||karbono|hiili|węgiel
 karbonodulokside||||carbon dioxide|dioxyde de carbone|dióxido de carbono|dióxido de carbono|двуокись углерода||二氧化碳||||प्रांगार द्विजारेय|||||karbona dioksido|hiilidioksidi|dwutlenek węgla
 karde|||eng:card, jpn: カード (kādo), swa: kadi, zho:卡 (kǎ), deu:Karte, may:kartu, fra:carte|card|carte|tarjeta (carta)|carta|карта||牌|カルタ|||ताश||kartu|karata||karto|kortti|karta
+kariba||||approach (get near)||axproximarse||||||||||||||lähestyä|
+karibe||||proximity||proximidad|||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: bon! :)

File: pandunia-lekse.csv [hunk 7]
```diff
@@ -1341,6 +1362,7 @@ krize|||eng:crisis, fra:por:crise, deu:Krise, rus:кризис (krizis), spa:cri
 krizi||||critical (pertaining to crisis)||crítico (fundamental)||||||||||||||kriittinen (kriisi-)|kryzysowy
 kromium|mate|Cr||chromium||cromo|||||||||||||kromo|kromi|chrom
 krote|||rus:крот (krot)|mole (burrowing animal)||topo||||||||||||||kontiainen (maamyyrä)|kret
+krusa||||cross||cruzar|||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: Good. Need additional English translation to notify that this is a verb, like "cross (to go across)"

File: pandunia-lekse.csv [hunk 8]
```diff
@@ -1878,6 +1904,8 @@ nordi amerike||||North America||Norteamérica||||||||||||||Pohjois-Amerikka|
 nordi cosenia|dex|KP||North Korea (Democratic People’s Republic of Korea)||Corea del Norte||||||||||||||Pohjois-Korea|
 nordi kipris|dex|||Northen Cyprus||Chipre del Norte||||||||||||||Pohjois-Kypros|
 nordi makedonia|dex|MK||Republic of North Macedonia||Macedonia del Norte||||||||||||||Makedonia|Republika Macedonii, Macedonia
+nordia||||latitude||latitud|||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: sudonordia?

File: pandunia-lekse.csv [hunk 9]
```diff
@@ -2133,6 +2161,7 @@ prasedim|mate|Pr||praseodymium||praseodimio||||||||||||||praseodyymi|
 prate|||eng:practice, fra:pratique, por:prática, tur:pratik, pol:praktyka, hin:प्रथा (prathā)|practice (actuality)||práctica||||||||||||||käytäntö (pragmatia)|praktyka, aktualność
 prati||||practical (pragmatic)||práctico||||||||||||||käytännöllinen (pragmaattinen)|practyczny, pragmatyczny
 preda|coverb||rus:п(е)ред (p(e)red), pol:przed, eng:spa:por:pre-, fra:pré-|precede (come before)||preceder (anteceder)|||||||||||||antaŭiri|edeltää (tulla ennen)|poprzedzić, poprzedzać
+prede||||lead-up|||||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: Beforeness? Front?

File: pandunia-lekse.csv [hunk 10]
```diff
@@ -2178,7 +2207,10 @@ radium|mate|Ra||radium||radio|||||||||||||radiumo|radium|rad
 radometer||||radiometer||radiómetro||||||||||||||säteilymittari|radoiometr
 radon|mate|Rn||radon||radón|||||||||||||radono|radon|radon
 raite|||eng:right|right hand side||lado derecho||||||||||||||oikea puoli|prawa strona
+raiti||||right hand||derecho|||||||||||||||
 raitiste||||rightist (right-winger)||derechista||||||||||||||oikeistolainen|prawicowiec (prawak)
+raito||||rightward||a la derecha|||||||||||||||
+raitolefti||||horizontal (lateral)||horizontal|||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: no malo... me fikra mome abla fata lekse "horizone" e "horizoni".

File: pandunia-lekse.csv [hunk 11]
```diff
@@ -2228,7 +2261,10 @@ reste|||fra:reste, eng:deu:Rest, pol:reszta, spa:por:resto|rest (remainder)|rest
 restoran||||restaurant||restaurante|||||||||||||restaŭracio|ravintola|restauracja
 restu||||remain (stay)||quedarse||||||||||||||jäädä jäljelle|zostać, zostawać
 retra||||return (go back)|retourner (revenir)|volver|voltar (retornar)|возвращаться||返回 (回来)||||लौटना|||||reiri|palata|wrócić, wracać, wycofać się, wycofywać się
-retre||||return (way back)||vuelta|||||||||||||reiro|paluu|
+retre||||back||dorso|||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: me fikra asli tarje si sahi. "rerurn" e "back of the body" si ali xey.

File: pandunia-lekse.csv [hunk 12]
```diff
@@ -2533,8 +2571,13 @@ tajikia|dex|TJ||Tajikistan||Tayikistán||||||||||||||Tadžikistan|Tadżykistan
 taksa||||collect fees or taxes||gravar (cobrar tasas)||||||||||||||veloittaa (verottaa)|
 takse|||eng:tax, por:taxa, spa:tasa, hin:टैक्स (ṭEks)|tax (fee, fare)||tasa (precio, impuesto, tarifa)||||||||||||||taksa (vero)|podatek, opłata
 taksokare||||taxi||taxi (cab)||||||||||||||taksi|taxi, taksówka
+tala||||lower (put down)||bajar|||||||||||||||
 tale||||bottom||fondo||||||||||||||pohja|dół, spód
+tali||||low||bajo|||||||||||||||
 talium|mate|Tl||thallium||talio|||||||||||||talio|tallium|tal
+talo||||down||abajo|||||||||||||||
+talodarja||||downgrade||degradar|||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: darjomala?

File: pandunia-lekse.csv [hunk 13]
```diff
@@ -2566,8 +2609,9 @@ tehne|||ell:τεχνική (tehniki), rus:техника (tehnika), fra:eng:tech
 tehnetium|mate|Tc||technetium||tecnetio|||||||||||||teknecio||technet
 tehnokratia||||technocracy||tecnocracia||||||||||||||teknokratia|technokracja
 tehnolojia||||technology (study of techniques)|technologie|tecnología||технология||工艺学|||||||||teknologio|tekniikka (teknologia)|technologia, nauka o technikach
-tele||||distance||distancia|||||||||||||malproksimeco|etäisyys (kaukaisuus)|dystans
+tele||||distance (far away)||lo lejos|||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: esperanti e suomi tarje paso sahi.

File: pandunia-lekse.csv [hunk 14]
```diff
@@ -2586,6 +2630,8 @@ tenu||||belong|appartenir à|ser de||принадлежать||属于|||||||||ap
 teoria||||theory||teoría|||||||||||||teorio|teoria|teoria
 tepe|||eng:tape, swa:tepu, jpn:(tēpu), tur:teyp|tape|ruban (bande)|cinta adhesiva||лента|||||||||||bendo|teippi (nauha)|taśma
 terbium|mate|Tb||terbium||terbio|||||||||||||terbio||terb
+terka||||tilt (skew)||inclinar|||||||||||||||
+terki||||oblique (askew, tilted)|oblique (de travers)|oblicuo (torcido)|oblíquo (torto)|косой||斜的|斜め||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: leksaslia si ke? me fikra "klina" e "klini" si bon cuzable ca rusi, engli e latini baxe.

File: pandunia-lekse.csv [hunk 15]
```diff
@@ -2764,14 +2810,16 @@ wate||||watt||vatio|||||||||||||vato|watti|wat
 waw|||zho:哇 (wā), eng:wow|wow! (gee!)||guau (vaya)||вау!|||||||||||ŭaŭ!|oho (vau)|wow!, łoł!, jej!
 waxa|||eng:wash, deu:waschen, zho:洗 (xǐ)|wash||lavar||мыть|||||||||||lavi|pestä|prać, umyć, myć
 waxomaxin||||washing machine||lavadora||стиральная машина|||||||||||lavmaŝino|pesukone|pralka
-wayi||||outer (external)||exterior|||||||||||||ekstera|ulko- (ulkoinen)|zewnętrzny
-waya||||oust (expel)||expulsar||||||||||||||poistaa|
+waya||||be outside of (go outside)||estar fuera de|||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv]: me fikra -a mena fati (active) xey e -u mena beyi (passive) xey.

File: pandunia-lekse.csv (hunk 1 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: okey. e nordia/sudonordia sami.

File: pandunia-lekse.csv (hunk 2 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: engli si tena yi lekse, a espani no. me no jana le ko uzi sa pandunia. ablo mome no nida le.

File: pandunia-lekse.csv (hunk 4 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: hem. darjobona bon.

File: pandunia-lekse.csv (hunk 5 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: m, me fila "netodona".

File: pandunia-lekse.csv [hunk 16]
```diff
@@ -1073,6 +1087,10 @@ jeste||||gesture||gesto||||||||||||||ele|gest
 jibraltar|dex|GI||Gibraltar||Gibraltar||||||||||||||Gibraltar|Gibraltar
 jibutia|dex|DJ||Djibouti||Yibuti||||||||||||||Djiboutia|Dżibuti
 jina|||zho:進 (jìn), jpn:進 (shin, jin), kor:진 (jin), vie:tiến|advance (progress, enter)||avanzar (entrar, adelantar, progresar)||||||||||||||edetä (edistyä)|posuwać się do przodu (dokonywać postępu, wejść)
```
[the remark this answers is not in the document]
Comment (jkunimune15) [reply pandunia-lekse.csv]: Okay. I think we should have a root for fore/front, though. me wil testa eureka bon nasomedi lekse.

File: pandunia-lekse.csv (hunk 8 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: we okey.

File: pandunia-lekse.csv (hunk 9 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: Actually, perhaps "prede" could mean front instead of "jine".

File: pandunia-lekse.csv (hunk 10 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: me fikra kombo lekse max bon, putongo.

File: pandunia-lekse.csv (hunk 11 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: aga "prede" mena "front", "bade" abla mena "back".

File: pandunia-lekse.csv [hunk 17]
```diff
@@ -2533,8 +2571,13 @@ tajikia|dex|TJ||Tajikistan||Tayikistán||||||||||||||Tadžikistan|Tadżykistan
 taksa||||collect fees or taxes||gravar (cobrar tasas)||||||||||||||veloittaa (verottaa)|
 takse|||eng:tax, por:taxa, spa:tasa, hin:टैक्स (ṭEks)|tax (fee, fare)||tasa (precio, impuesto, tarifa)||||||||||||||taksa (vero)|podatek, opłata
 taksokare||||taxi||taxi (cab)||||||||||||||taksi|taxi, taksówka
+tala||||lower (put down)||bajar|||||||||||||||
 tale||||bottom||fondo||||||||||||||pohja|dół, spód
+tali||||low||bajo|||||||||||||||
 talium|mate|Tl||thallium||talio|||||||||||||talio|tallium|tal
+talo||||down||abajo|||||||||||||||
+talodarja||||downgrade||degradar|||||||||||||||
+talolada||||download||descargar|||||||||||||||
```
[the remark this answers is not in the document]
Comment (jkunimune15) [reply pandunia-lekse.csv]: te fikra ke tema "netogeta"?

File: pandunia-lekse.csv (hunk 14 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: upse; me memula leksalsia. me fikra ye pol medonasi.
spa:torcido, por:torto, hin:तिरछा (tirachha), ben:তেরছা (tērachā), pan:तिरकस (tirakasa)

File: pandunia-lekse.csv (hunk 15 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: si, a we no maca "nica", "bada", o "cirka". me fikra bizari sate da su ali dixe alo funca.

File: pandunia-lekse.csv (hunk 3 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: okey. me aga mome no nida le.

File: pandunia-lekse.csv (hunk 2 above)
Comment (barumau) [reply pandunia-lekse.csv]: me fikra "eastward" = dongodixo o pa donge.

File: pandunia-lekse.csv (hunk 3 above)
Comment (barumau) [reply pandunia-lekse.csv]: gawi = high, gawo = highly

File: pandunia-lekse.csv (hunk 5 above)
Comment (barumau) [reply pandunia-lekse.csv]: okey.

File: pandunia-lekse.csv (hunk 16 above)
[the remark this answers is not in the document]
Comment (barumau) [reply pandunia-lekse.csv]: si, mome nida wi leksomule.

File: pandunia-lekse.csv (hunk 9 above)
Comment (barumau) [reply pandunia-lekse.csv]: si :)

File: pandunia-lekse.csv (hunk 10 above)
Comment (barumau) [reply pandunia-lekse.csv]: a raite e lefte no si horizoni. lole si dixe da bode.

File: pandunia-lekse.csv (hunk 11 above)
Comment (barumau) [reply pandunia-lekse.csv]: si.

File: pandunia-lekse.csv (hunk 17 above)
[the remark this answers is not in the document]
Comment (barumau) [reply pandunia-lekse.csv]: le bon.

File: pandunia-lekse.csv (hunk 14 above)
Comment (barumau) [reply pandunia-lekse.csv]: me fikra ren no abla jana "terki" ca "torto" o "tirachha".

File: pandunia-lekse.csv (hunk 15 above)
Comment (barumau) [reply pandunia-lekse.csv]: me bixu fikra le...

File: pandunia-lekse.csv (hunk 3 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: a, bon argum. te rayisa me.

File: pandunia-lekse.csv (hunk 10 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: me gara te sahi. me fikra ko fiziker; pa me si 3 dixe: gawotali (vertical), raitolefti (horizontal), e predobadi (fordinal). a pa normi ren, we da logu te.

File: pandunia-lekse.csv (hunk 14 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: m. eska te fikra "terci" max bon?

File: pandunia-lekse.csv (hunk 15 above)
Comment (jkunimune15) [reply pandunia-lekse.csv]: okey. me wil hafiza le sa yi sate.

**Pull request merged**